content-scripts/pages/components_page.tsx

```diff
@@ -208,6 +209,45 @@ export function createComponentsPage() {
                         ]}
                     />
                 </Component>
+
+                {/* Accordion */}
+                <Component
+                    name="Accordion"
+                    description="A collapsible component that can expand to show or hide content."
+                    code={`
+<Accordion
+  id="example-accordion"
+  title="Expandable Content"
+  data={<p>This is the content of the accordion</p>}
+  max_size={200}
+/>
+                    `}
+                >
+                    <Accordion
+                        id="example-accordion"
+                        title="Expandable Content"
+                        data={[<p>This is the content of the accordion</p>,
+                            <p>This is the content of the accordion</p>,
+                            <Table
+                                name="my_table"
+                                headers={[
+                                    ["Component", "Component"],
+                                    ["Description", "Description"],
+                                    ["Status", "Status"],
+                                ]}
+                                data={[
+                                    [
+                                        "Button",
+                                        "A button that can be clicked",
+                                        "In progress",
+                                    ],
+                                    ["Input", "A text input field", "Complete"],
+                                ]}
+                            />
+                        ]}
+                        max_size={200}
+                    />
+                </Component>
```

Comment:
You can try to use the following syntax for the data property.

```html
<Accordion>
   <p>This is a content of the according</p>
</Accordion>
```

Check an [example](https://github.com/ConnorJamesLow/texsaur/blob/main/examples/vite-project/example.jsx) from texsaur repo.